Feature/h13 panel reservas usuario

carolinasosads opened the pull request.

carolinasosads committed:
H13 - Comienzo de requerimiento, commiteado para poder hacer rebase sobre otra rama

# Conflicts:
#	WinniElectricidad-Frontend/src/components/App/App.jsx

carolinasosads committed:
H13 - Comienzo de requerimiento, commiteado para poder hacer rebase sobre otra rama

# Conflicts:
#	WinniElectricidad-Frontend/src/components/App/App.jsx
#	WinniElectricidad-Frontend/src/pages/User/PanelReservas/PanelReservasCliente.jsx
#	WinniElectricidad-Frontend/src/pages/User/PanelReservas/ReservaDetailModalCliente.jsx

carolinasosads committed:
H13 - commits para crear una nueva rama

carolinasosads committed:
H13 - Modificacion en metodo de aprobar reservas, creacion de panel para usuarios y logica

carolinasosads committed:
H13 - Cambios en la bd para que el admin pueda mover una reserva a una fecha con horario cancelado

carolinasosads committed:
H13 - No permite sugerir fecha a menos de 48 horas tanto para el cliente como para el admin

sofvillanueva commented:
Deberiamos hacer algun test del nuevo método de ObtenerMisReservasClienteConDetalle

carolinasosads committed:
H13-Cambios en como se guarda el presupusto, ahora se usa la clase pago. Modificacion en como se crea la hora de las reservas. Fixes genericos en base a los calendarios

carolinasosads committed:
H13-Mapeo correcto de los comentarios de la reserva

carolinasosads committed:
H13-Comentarios copilot

# Conflicts:
#	WinniElectricidad-Frontend/src/pages/Admin/PanelReservas/PanelReservas.jsx

carolinasosads committed:
H13 - Implementacion calendario mobile

carolinasosads committed:
H13 - push de implementacion de calendario

carolinasosads committed:
H13-Test Large
